lldb/test/API/commands/command/script/exception/TestCommandException.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
A bit surprised we don't have a scripted commands test that this could be added to instead of making a new file.

Reply:
What are the issues with a new file?